kedro/io/partitioned_data_set.py

```diff
@@ -216,8 +219,9 @@ def _normalized_path(self) -> str:
 
     @cachedmethod(cache=operator.attrgetter("_partition_cache"))
     def _list_partitions(self) -> List[str]:
+        dataset_is_versioned = self._dataset_config.get(VERSION_KEY, False)
```

Comment:
```suggestion
        dataset_is_versioned = self._dataset_config.get(VERSIONED_FLAG_KEY, False)
```
Should this actually be `VERSIONED_FLAG_KEY`, a la https://github.com/quantumblacklabs/kedro/blob/774a29d925a4331e70743155963cd9b643a6bd8c/kedro/io/core.py#L435? I don't think it matters functionally, more from a readability perspective.

Reply:
JK, https://github.com/quantumblacklabs/kedro/blob/develop/kedro/io/core.py#L430 pops `VERSIONED_FLAG_KEY` out of the parsed config.